its/ruling/src/test/resources/regex-examples/java-S5994.json

```diff
@@ -0,0 +1,5 @@
+{
+'org.regex-examples:regex-examples:src/main/java/org/regex/examples/RegexDatabase5.java':[
+135,
```

Comment:
Seems to be a false-positive, I will investigate

Reply:
The possessive quantifier check should be changed to not apply to repetitions with a fixed upper bound.